### PR TITLE
feat(Tabs): [FX-2058] update tabs to the latest design

### PR DESCRIPTION
**Pull request by teimurjan**
[FX-2058](https://toptal-core.atlassian.net/browse/FX-2058)

### Description

- Updated disabled color
- Replaced legacy visual tests with the Happo ones
- Added more functionality-oriented unit tests.

### How to test

- FIXME: Add the steps describing how to verify your changes

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| Insert screenshots or screen recordings | Insert screenshots or screen recordings |

### Review

- [ ] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/api-principles.md)
- [ ] Annotate all `props` in component with documentation
- [ ] Create `examples` for component
- [ ] Ensure that deployed demo has expected results and good examples
- [ ] Ensure that tests pass by running `yarn test`
- [ ] Ensure that visuals tests pass by running `yarn test:visual`. If not - check the documentation [how to fix visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md#fixing-broken-visual-tests-inside-a-pr)
- [ ] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

### CODE REVIEW

**Comment by teimurjan** [comment]
@toptal-bot run visual

**Review by vedrani — APPROVED** [review]

File: cypress/component/Tabs.spec.tsx [hunk 1]
```diff
@@ -1,50 +1,104 @@
...
-          <Tabs.Tab label='Label' data-testid='tab-4' />
-        </Tabs>
+      <div style={{ width }}>
+        <TestTabs data-testid='tabs'>
+          {Array.from({ length: 5 }).map((_, index) => (
+            <Tabs.Tab
+              disabled={disabledIndicies?.includes(index)}
+              key={index}
+              label='Label'
+              data-testid={getTabTestId(index)}
```
Comment (havenchyk) [inline cypress/component/Tabs.spec.tsx:52]: nit: `tab-${index}`

File: cypress/component/Tabs.spec.tsx [hunk 2]
```diff
@@ -1,50 +1,104 @@
...
+              icon={
+                withIconIndicies?.includes(index) ? <TestIcon /> : undefined
```
Comment (havenchyk) [inline cypress/component/Tabs.spec.tsx:54]: What do you think about having the default value `[]` for `withIconIndicies`?

File: packages/picasso/src/Tabs/test.tsx [hunk 3]
```diff
@@ -1,52 +1,111 @@
...
 
 const renderTabs = (
-  children: ReactNode,
-  props: OmitInternalProps<Props, 'children'>
+  tabs: { label: string; value?: string; disabled?: boolean }[],
+  { value, onChange }: OmitInternalProps<Props, 'children'>
 ) => {
-  const { value } = props
+  const content = tabs.map((tab, index) =>
+    index + 1 === value || tab.value === value ? (
```
Comment (havenchyk) [inline packages/picasso/src/Tabs/test.tsx:12]: can you simplify it please?

like

```
content = null

if () {
  content = ...
}
```

would be easier to read

**Review by havenchyk — APPROVED** [review]
LGTM, just small notes left

**Pull request merged**